src/text_formatter/text_formatter.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
I don't think you need these imports, nor does the file need to be a `tsx` file. Make it a `ts` file instead.